### PR TITLE
Welcome Bot Message

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: samhita-alla <aallasamhita@gmail.com>

### CODE REVIEW

**Comment by samhita-alla** [comment]
I believe these changes should suffice. Please do let me know if there's anything else to be modified.

**Comment by kumare3** [comment]
Nothing to be modified, but you should try out the change on couple repos and if they work then we are good to go. Just one comment and then +1

**Comment by samhita-alla** [comment]
I tried validating welcomebot in the flyteidl repo, and it didn't work. I'm not sure if it's because of the fork. So, I've created a new repo and it worked fine.

image

The best way to validate this is to merge it into the boilerplate repo and then verify it in a couple of repos.

**Review by kumare3 — DISMISSED** [review]

File: boilerplate/flyte/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Comment to be posted on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thank you for opening this pull request! 🙌
+
+  These tips will help get your PR across the finish line:
+  - Refer to the [issue template](https://github.com/flyteorg/flyte/tree/master/.github/ISSUE_TEMPLATE).
+  - Sign off your commits (Reference: [DCO Guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md)).
+  - Add a short description of your PR.
```
[the remark this answers is not in the document]
Comment (samhita-alla) [reply boilerplate/flyte/config.yml]: Done.

**Review by kumare3 — APPROVED** [review]

**Pull request merged**